src/modules/personenkontext/api/personenkontext.response.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { faker } from '@faker-js/faker';
+import { PersonenkontextResponse } from './personenkontext.response.js';
+import { LoeschungResponse } from '../../person/api/loeschung.response.js';
+import { Rolle } from '../domain/personenkontext.enums.js';
+
+describe('PersonenkontextResponse', () => {
+    describe('constructor', () => {
+        describe('when setting loeschung prop', () => {
+            it('should create instance of LoeschungResponse', () => {
+                // Arrange
+                const props: PersonenkontextResponse = {
+                    id: faker.string.uuid(),
+                    mandant: faker.string.uuid(),
+                    organisation: {
+                        id: faker.string.uuid(),
+                    },
+                    rolle: Rolle.LEHRENDER,
+                    revision: faker.string.uuid(),
+                    loeschung: {
+                        zeitpunkt: faker.date.recent(),
+                    },
+                };
+
+                // Act
+                const result: PersonenkontextResponse = PersonenkontextResponse.new(props);
+
+                // Assert
+                expect(result.loeschung).toBeInstanceOf(LoeschungResponse);
+            });
+        });
+
+        describe('when setting loeschung prop to undefined', () => {
+            it('should not create instance of LoeschungResponse', () => {
+                // Arrange
+                const props: PersonenkontextResponse = {
+                    id: faker.string.uuid(),
+                    mandant: faker.string.uuid(),
+                    organisation: {
+                        id: faker.string.uuid(),
+                    },
+                    rolle: Rolle.LEHRENDER,
+                    revision: faker.string.uuid(),
+                    loeschung: undefined,
+                };
+
+                // Act
```

Comment:
is this comments necessary?

Reply:
No, but since we don't use a setup function I put these comments in for a better visual separation between the different sections of the test and a better readability.